File: 485-max-consecutive-ones/max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int count=0;
        int max=count;
        for(int i=0; i<nums.size(); i++){
            if(nums[i]==1)    count++;
            else   count=0;
            if(count>=max)   max=count;
        }
        return max;
    }
};

File: 485-max-consecutive-ones/max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int count =0;
        int maxCount =0;
        for(int e: nums){
            if(e==1)    count+=1;
            else count=0;
            maxCount = Math.max(maxCount,count);
        }
        return  maxCount;
    }
}